### ISSUE TITLE
Using require('ews-javascript-api') throws "Error: Cannot find module '../Extensionmethods'"

### ISSUE THREAD

**Issue by losmescaleros** [gautamsi/ews-javascript-api#16]
Using Node v4.4.0, the "require" statement throws this error. You can also see this on the [npm package site](https://tonicdev.com/npm/ews-javascript-api).

Full output:
```
vagrant@vagrant-ubuntu-trusty-64:~/test$ node app.js 
module.js:327
    throw err;
    ^

Error: Cannot find module '../Extensionmethods'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/vagrant/test/node_modules/ews-javascript-api/js/PropertyDefinitions/PropertyDefinitionBase.js:6:26)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
```

**Comment by gautamsi** [comment]
this was fixed by #14 but I did not publish new version yet,

you can change `require("../Extensionmethods")` with `require("../ExtensionMethods")` in file node_modules/ews-javascript-api/js/PropertyDefinitions/PropertyDefinitionBase.js

when I push new version it will be fixed.

**Comment by gautamsi** [comment]
let me know if you are still facing this issue.
